app/models/ability.rb

```diff
@@ -88,10 +88,10 @@ def can_assign_ownership(user)
       end
     end
 
-    def can_delete_membership(team_member, user)
-      can :delete_membership, Team, Team do |team|
+    def can_delete_membership(user)
+      can :delete_membership, Team, Team do |team, team_member|
```

Comment:
Way better approach than the one before with the second parameter in `initialize` :+1: